editor/editor_help.cpp

```diff
@@ -1831,6 +1831,15 @@ void EditorHelp::_update_doc() {
 			class_desc->push_indent(1);
 			if (!cd.properties[i].description.strip_edges().is_empty()) {
 				_add_text(DTR(cd.properties[i].description));
+				// Add copy note to built-in properties returning Packed*Array.
+				if (!cd.is_script_doc &&
```

Comment:
Restricting to built-in classes at the moment, as from what I can tell at least for GDScript the data isn't copied like this but remains shared, but will investigate further as well